<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Riden Trails</string>
    </property>
   </widget>
   <widget class="QPushButton" name="home">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="searchText">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>660</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>640</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add trail to list</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>670</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QListWidget" name="trailsRiden">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>501</width>
      <height>561</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <family>Rockwell Condensed</family>
       <pointsize>20</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <family>Rockwell Condensed</family>
       <pointsize>20</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <family>Rockwell Condensed</family>
       <pointsize>20</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <family>Rockwell Condensed</family>
       <pointsize>20</pointsize>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <family>Rockwell Condensed</family>
       <pointsize>20</pointsize>
      </font>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>750</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Application by Brendan Cook</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
